CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# dbt_snapchat_ads v0.4.0
+PR [#11](https://github.com/fivetran/dbt_snapchat_ads/pull/11) applies the Ad Reporting V2 updates:
+## 🎉 Feature Enhancements 🎉
+
+- Addition of the following new end models. These models were added to provide further flexibility and ensure greater accuracy of your Snapchat Ads reporting. Additionally, these new end models will be leveraged in the respective downstream [dbt_ad_reporting](https://github.com/fivetran/dbt_ad_reporting) models.
+  - `snapchat_ads__ads_report`
+    - Each record in this table represents the daily performance at the ads level.
+  - `snapchat_ads__creative_report`
+    - Each record in this table represents the daily performance at the campaign level.
+  - `snapchat_ads__url_report`
+    - Each record in this table represents the daily performance at the url level.
+
+- Applies README standardization updates
+- Introduces the identifier variable for all source models
+- Casts all timestamp fields using dbt_utils.type_timestamp() and rounds all monetary fields. 
+- Introduces metrics passthrough capability 
+
+## 🚨 Breaking Changes 🚨
+- Changes `snapchat_schema` and `snapchat_database` variable names to `snapchat_ads_schema` and `snapchat_ads_database` 
+- Updates model names to prefix with `snapchat_ads` and removed the `ad_adapter` model and dependencies on it
```

Comment:
Small request to move this to the top of the release notes. This way the breaking changes are the first thing people see before upgrading.